src/Tooltip2.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {Children, useEffect, useRef} from 'react'
+import Box from './Box'
+import {BetterSystemStyleObject, merge, SxProp} from './sx'
+import {useId} from './hooks/useId'
+import {isFocusable} from '@primer/behaviors/utils'
+
+export type Tooltip2Props = {
+  direction?: 'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w' | 'nw'
+  text?: string
+  noDelay?: boolean
+  align?: 'left' | 'right'
+  wrap?: boolean
+  type?: 'label' | 'description'
```

Comment:
`type` is a new prop that I added and defaulted to `label` to prevent braking the existing usage.

Reply:
@broccolinisoup for `type` here, I bet we could have the default be `type = 'label'` as that seems to be the semantics from:

> Attention: we use aria-label for tooltip contents. However, aria-label replaces the text content of an element in screen readers, so only use Tooltip on elements with no existing text content.
> [Docs](https://primer.style/react/Tooltip)

Would this help with releasing this in a minor release? We could then make this required, if appropriate, in the next major.

Let me know if that makes sense! 

---

Side-note, the link for usage points to PVC. Is there a query link that you were sharing? Would love to check it out!